File: Phyton/Vetor/maiorPosicao.py
n = int(input("Quantos numeros voce vai digitar? "))

vet = [0 for x in range(n)]
maior = 0
posicao = 0
for i in range(n):
    vet[i] = float(input("Digite um numero: "))

for i in range(1, n):
    if vet[i] > vet[i-1]:
        posicao = i

print(f"MAIOR VALOR = {vet[posicao]:.1f}")
print(f"POSICAO DO MAIOR VALOR = {posicao}")

File: C/Vetor/maior_posicao.c
#include<stdio.h>
int main()
{
int n, i, pos=0;
double maior=0;

printf("Quanto numeros voce vai digitar?");
scanf("%d", &n);
    double vet[n];

        for(i=0; i<n; i++){
            printf("Digite um numero:");
            scanf("%lf", &vet[i]);
              if(vet[i] > maior){
                maior = vet[i];
                pos = i;
              }

        }

        printf("\nMAIOR VALOR = %.2lf\n", maior);
        printf("POSICAO DO MAIOR VALOR = %d", pos);





}
